internal/plugin.go

```diff
@@ -844,17 +848,23 @@ func (p *Plugin) createPolicy(policyConf *types.PolicyConfig) error {
 		return err
 	}
 
+	annotations := map[string]string{
+		"policy.open-cluster-management.io/categories": strings.Join(policyConf.Categories, ","),
+		"policy.open-cluster-management.io/controls":   strings.Join(policyConf.Controls, ","),
+		"policy.open-cluster-management.io/standards":  strings.Join(policyConf.Standards, ","),
+	}
+
+	for k, v := range policyConf.PolicyAnnotations {
+		annotations[k] = v
```

Comment:
If a policy uses the default `PolicyAnnotations`, could you please create a copy of that in the `applyDefaults` method? That way, the implementation of `applyDefaults` does not need to be accounted for in `createPolicy`.

Reply:
its not what you mean, its already copy the DefaultPolicyAnnotation into the PolicyAnnotation 
https://github.com/stolostron/policy-generator-plugin/pull/44/files#diff-6e087f8d6059d270802616bc8a93ff47320e6302233e7ca9cf317b3c7054c8abR389

Reply:
@serngawy that copies the references but not the map itself. See this as an example:
https://riptutorial.com/go/example/9834/copy-a-map

Reply:
done